### ISSUE TITLE
map auto center

### ISSUE THREAD

**Issue by tacotonacho** [smasherprog/EqTool#103]
please add auto centering to the map when /loc is called

**Comment by smasherprog** [comment]
Yeah i havent had time to get to this, but its for sure on my to do list :)

**Comment by nem8** [comment]
Would be extremely helpfull :) Appreciate all the work on EqTool so far tho!

**Comment by smasherprog** [comment]
This might be done today, testing now

**Comment by smasherprog** [comment]
A Follow you has been added. Not quite Center on you, but its very close now.
